eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -357,6 +357,100 @@ function New-MgmtPackageFolder() {
     return $projectFolder
 }
 
+function New-CADLPackageFolder() {
+    param(
+        [string]$service,
+        [string]$namespace,
+        [string]$sdkPath = "",
+        [string]$cadlInput ="",
+        [string]$outputJsonFile = "output.json"
```

Comment:
This should be a full path. You could use `"$PWD\output.json"`, for example. Don't require pushing and popping paths as I noted above. It's fragile.

Reply:
uddate